Move `ServiceAccountPatching` feature gate to GA (#6869)
strimzi/strimzi-kafka-operator: documentation/modules/operators/ref-operator-cluster-feature-gates.adoc

== ServiceAccountPatching feature gate

> The `ServiceAccountPatching` feature gate has moved to GA stage in Strimzi 0.30 and is permanently enabled without the possibility to disable it.

Reviewer: ```suggestion
The `ServiceAccountPatching` feature gate has moved to GA, which means it is now permanently enabled and cannot be disabled.
```

> The Cluster Operator now always reconciles service accounts and updates them when needed.

Reviewer: ```suggestion
With `ServiceAccountPatching` enabled, the Cluster Operator always reconciles service accounts and updates them when needed.
```

> For example, when you change the desired service account labels or annotations using the `template` property of our custom resources, the operator will automatically update them on the existing service account resources.

Reviewer: ```suggestion
For example, when you change service account labels or annotations using the `template` property of a custom resource, the operator automatically updates them on the existing service account resources.
```
